examples/dataproc-persistent-history-server/terraform/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "project" {
+    description = "Project ID for your history cluster"
+}
+
+variable "history-server" {
+    description = "Cluster ID for the Spark / YARN history server"
+}
+
+variable "history-region" {
+    description = "GCP Compute Region for your history server and bucket"
+    default     = "us-central1"
+}
+
+variable "history-bucket" {
+    description = "The bucket name for your YARN logs and Spark event logs"
+}
+
+variable "network" {
+    description = "The network that your hadoop clusters should use"
```

Comment:
Use or create? "Use" implies an existing network, but this code actually creates it.